cyclonedds/idl/_xt_builder.py

```diff
@@ -974,9 +974,19 @@ def _xt_minimal_struct_member(cls, entity: Type[IdlStruct], name: str, _type: ty
 
     @classmethod
     def _xt_complete_struct_member(cls, entity: Type[IdlStruct], name: str, _type: type) -> xt.CompleteStructMember:
+        #Reserved Python keywords support (Issue 105)
+        realName=name
```

Comment:
This needs a little bit of formatting and removing the duplicate dict access. I suggest the following:

````python
# Reserved Python keywords support (#105)
annotations = get_idl_field_annotations(entity).get(name)
real_name = annotations["name"] if annotations and "name" in annotations else name

Reply:
Hello @thijsmie , thank you for your fast answer.

About "we can also resolve types dynamically from the network using the XTInterpreter", How do we have to do this? How do we have to write the IDL file for the method XTInterpreter._make_complete_struct or XTInterpreter._make_complete_union to be executed during the translation from IDL to Python ?

If you give us more information about how to reproduce this conversion, we can try to implement the change ourselves as we have done with the rest of cases.


About your suggestion "annotations = get_idl_field_annotations(entity).get(name) ...", you are right, your implementation is smarter, we will change it.